shinhye/db/migrations/20230127012509_create_likes_table.sql

```diff
@@ -0,0 +1,15 @@
+-- migrate:up
+CREATE TABLE likes (
+    id INT NOT NULL AUTO_INCREMENT PRIMARY KEY,
+    like_user_id INT NOT NULL,
+    liked_post_id INT NOT NULL,
```

Comment:
컬럼명 수정해주세요!
`like_`로 작성한다면 `like` 테이블을 조회할 때 `likes_like_user_id`로 조회를 할 것입니다.

그럴 필요 없이 `like_user_id`로 바로 접근할 수 있도록 수정하면 좋을 것 같습니다.

해당 부분과 관련된 부분 모두 수정해주세요!
```suggestion
    user_id INT NOT NULL,
    post_id INT NOT NULL,
```

Reply:
넵 반영했습니다!